WordPress/Classes/Services/Stories/WPMediaPicker+MediaPicker.swift

```diff
@@ -161,38 +165,50 @@ extension WPMediaAsset {
     func videoURLPublisher() -> AnyPublisher<URL, Error> {
         return videoAssetPublisher().tryMap { asset -> AnyPublisher<URL, Error> in
             let filename = UUID().uuidString
-            let url = try FileManager.default.url(for: .documentDirectory, in: .userDomainMask, appropriateFor: nil, create: true).appendingPathComponent(filename).appendingPathExtension("mp4")
+            let storiesURL = try MediaFileManager.cache.directoryURL().appendingPathComponent("Stories", isDirectory: true)
```

Comment:
It doesn't look like "Stories" is used anywhere else but are there plans to?  Maybe make this a constant if so?

Reply:
I decided to reuse this directory for the `saveDirectory` used in `StoryEditor` in b29af92. Mainly, I just wanted to keep these files separate in case we need to do any cleanup or rearranging later on.

`saveDirectory` is technically temporary (for before media is uploaded and receives its remote media ID), but we want to be in control so we aren't using the system temp directory.